PROPOSAL.md

```diff
@@ -0,0 +1,70 @@
+# COMP 4350 Proposal Team - Spendr
+
+## Project Summary and Vision
+
+Spendr is designed to empower individuals in monitoring and managing their expenses. The application provides intuitive features for individual expense tracking, analytics, budgeting, and seamless group expense splitting.
+
+## Core Features
+
+1. User Management
+2. Expense Entry and History
+3. Budget Management
+4. Expense Analytics
+5. Group Expense Splitting
+
+## User Stories 
+
+### Feature 1. User Management
+
+- **User Story**: As a potential user, I want to register an account with Spendr to start tracking my expenses.
+  - **Acceptance Criteria**:
+    - Can register with username and password.
```

Comment:
```suggestion
- **User Story**: As a new user, I want to register an account with Spendr to start tracking my expenses.
  - **Acceptance Criteria**:
    - Can register with using an email, username and password.
    - Can log in with username and email
```

Reply:
I feel like allowing user creation with emails is going to be harder to implement and test, since we're going to have to use third party services for sending/verifying emails, as well as mocking that service when testing.